src/components/DashboardPage/DashboardTask.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+
+import BtnTask from '@/components/common/BtnTask/BtnTask';
+import ScrollGradient from '@/components/common/ScrollGradient';
+
+interface DashboardTaskProps {
+	text: string;
+}
+
+function DashboardTask({ text }: DashboardTaskProps) {
```

Comment:
```suggestion
interface DashboardTaskProps {
	text: 'upcoming' | 'postponed' | 'inprogress';
}

function DashboardTask({ text }: DashboardTaskProps) {
	const DASHBOARD_TASK_TYPE = {
		UPCOMING: '다가오는 할 일',
		POSTPONED: '지연된 할 일',
		INPROGRESS: '진행중인 할 일',
	};
```

직접 텍스트로 제목을 받기보다는 요런 느낌으로 쓰면 오타 위험을 줄일 수 있을 것 같습니다!

Reply:
헐 레전드 이런 방법도 있군요 왜 생각하지 못했을까요,, 감사합니다~!!~!!!!